backlog-check-wip-limit

```diff
@@ -1,9 +1,39 @@
-#!/bin/sh -e
+#!/bin/bash -e
 redmine_api_key="${redmine_api_key:?"Need redmine API key"}"
 host="${host:-"https://progress.opensuse.org"}"
 query_id="${query_id:-400}"
 ticket_limit="${ticket_limit:-200}"
 wip_limit="${wip_limit:-10}"
 status="${status:-"In Progress"}"
-tickets=$(curl -s -H "X-Redmine-API-Key: $redmine_api_key" "https://progress.opensuse.org/issues.json?query_id=$query_id&limit=$ticket_limit" | jq -r '.issues | .[] | select(.status.name=="In Progress") | .id')
-test "$(echo "$tickets" | wc -l)" -le "$wip_limit"
+
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
+
+Checks the number of tickets in a redmine query against a "WIP-Limit".
```

Comment:
Please document at least the most important env vars and what happens if check fauils (exit code=1?)